core/baseline-core-v1.0-psd01.md

```diff
@@ -2786,113 +2795,722 @@ The document defines storage capability requirements in the following areas:
+* The communication between the services is captured using a packet capture system
+* The encryption algorithm(s) in the test meets the security requirements of this document.
+
+Test Steps:
+
+1. Start the communication between the two services in the test environment using some plain text test data which is encrypted/decrypted by each service.
+2. Capture the network traffic using the packet capture tool.
+3. Inspect the captured packets to confirm that all data transmitted between the nodes is encrypted, and not in plain text, and can be decrypted using the key(s) used for encryption.
+5. Repeat steps 1-3 for different types of data and communication scenarios (e.g., different data formats, different network topologies).
+
+Expected Results: All data transmitted between the nodes is encrypted (does not appear in plain text).
+
+The requirement is met, if all expected results are met. The requirement is not me, if any of the results are not meeting test expectations.
+
 #### **[R280]** 
 Data at rest in a BPI MUST be encrypted.
 
+[[R280]](#r280) Testability: 
+
+Preconditions: 
+
+* There is a BPI test environment that can store data
+* The encryption algorithm(s) in the test meets the security requirements of this document.
+
+Test Steps:
+
+1. Identify the location of the data at rest in the BPI.
+2. Verify that the data is encrypted and cannot be decrypted without the encryption/decryption key.
+3. Attempt to decrypt the encrypted data without the encryption/decryption key and verify that it is not possible.
+5. Repeat steps 1-3 for different types of data and storage scenarios (e.g., different data formats, different storage media).
+
+Expected Results: The data at rest in the BPI is encrypted and cannot be decrypted without the encryption/decrytion key.
+
+The requirement is met, if all expected results are met. The requirement is not me, if any of the results are not meeting test expectations.
+
 #### **[R281]**	
 BPI Storage arranged in a network MUST support pairwise key/identity relationships between storage nodes.
 
 *This is also known as a secure connection.*
 
+[[R281]](#r281) Testability: 
+
+Preconditions: 
+
+* There is a BPI test environment with at least two independent storage nodes that communicate with each other
+* The communication between the services is onfigured to support pairwise key/identity relationships
+* The encryption/decryption algorithm(s) in the test meets the security requirements of this document.
+
+Test Steps:
+
+1. Verify that each storage node has a unique identity and a corresponding public key.
+2. Establish a pairwise key/identity relationship between two storage nodes.
+3. Verify that the storage nodes can communicate securely using the established relationship.
+4. Attempt to access the data stored on one storage node from the other without using the established relationship and verify that it is not possible.
```

Comment:
```suggestion
4. Verify that the storage node does not provide access to the data stored except using the established relationship.
```

This is a re-word that I think still isn't quite right, but doesn't require proof of impossibility (because that's usually not feasible)